doc/X-OR_tutorial/X-OR_tutorial.rst

```diff
@@ -712,9 +712,9 @@ And this maximum is:
 
 If this is not clear enough you can take a look at the edge labels in the above graph.
 
-To choose a base for a tree we need to apply the previous algorithm to all the roots of tree.
+To choose a base for a tree we, need to apply the previous algorithm to all the roots of tree.
```

Comment:
Several roots for a tree ? Are we manipulating a tree, a graph, or a collection of trees ? That's not very clear.